packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_localizations.dart

```diff
@@ -1302,8 +1302,8 @@ abstract class UbuntuBootstrapLocalizations {
   /// No description provided for @restartWarning.
   ///
   /// In en, this message translates to:
-  /// **'You can continue testing {RELEASE} now, but until you restart the computer, any changes you make or documents you save will not be preserved.'**
-  String restartWarning(String RELEASE);
+  /// **'Restart to complete the installation or continue testing.\nAny changes you make not be saved.'**
+  String restartWarning(Object RELEASE);
```

Comment:
Hm, the generated function takes an argument even though it has been remove in the arb file. Is that because the non-"en" arb files haven't been updated accordingly? (In that case we might have that issue with a couple more l10n strings elsewhere).